griptape/events/text_chunk_event.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from attrs import define
+
+from griptape.events.base_chunk_event import BaseChunkEvent
+
+if TYPE_CHECKING:
+    from griptape.common import BaseDeltaMessageContent
+
+
+@define
+class TextChunkEvent(BaseChunkEvent):
+    @classmethod
+    def from_delta_message_content(cls, content: BaseDeltaMessageContent) -> TextChunkEvent:
+        from griptape.common import TextDeltaMessageContent
+
+        if isinstance(content, TextDeltaMessageContent):
+            return cls(token=content.text, index=content.index)
+
+        raise ValueError(f"Content is not an instance of TextDeltaMessageContent: {content.__class__.__name__}")
```

Comment:
Same as `ActionChunkEvent`. I don't think this factory method should exist on Events.

Reply:
was following the pattern i found [in the `BaseActionsSubtaskEvent`](https://github.com/griptape-ai/griptape/blob/dev/griptape/events/base_actions_subtask_event.py) and i liked it. what would be the difference between these two?

Reply:
That method appears to be unused. We just instantiate the events [normally](https://github.com/griptape-ai/griptape/blob/bf96755bbe52afef66ecd80fa682394abe498fe4/griptape/tasks/actions_subtask.py#L96-L107).